script/Arcueid/Day 7/07_00_ARC07_3F.txt

```diff
@@ -1122,7 +1122,7 @@ Explain yourself!
 -- Page 414, Offset 17606. Glued. Mods: @x.
 -- おまえの体が治りきるまで手伝ってやるって、そう言いたいんだよ、俺は！」
 // Leading space for glue
- Until your body gets better, I'm going to help you out!"
+ until your body gets better, I'm going to help you out!"
```

Comment:
American English asks for a capital after a colon, but it's up to you. In British English, we wouldn't.

Reply:
American English grammatical style guides are quite split on this. MLA, Chicago, and APA have strongly opposing views regarding whether or not single, independent clauses introduced by colons should be capitalized.

Here is my thinking: the closest analogous case is an independent clause following a semicolon. Those aren't capitalized and, as with the colon, they're part of the same _sentence_. As such, I don't think phrases should be capitalized following colons. That's also what Chicago says and I tend to agree with Chicago the most anyways.

Reply:
Actually I don't think that those styles are applicable to the situation. I believe the Parentheses in the previous line and this line where messed up and We should treat this as a Seperate peice of quoted text. Of course I don't speak Japanese but from the context this line and the last one should look like this:
("Here's what I want to say:)->(Here's what I want to say:) 
(Until your body gets better, I'm going to help you out!")->("Until your body gets better, I'm going to help you out!")

Reply:
This isn't two separate lines in game, it's glued. All of it is voiced, and all of it is within the 「」. I have no strong preference on capitalization either way though.

Reply:
if its the same line then no capitalization.